tempest/api/workloadmgr/restore/test_restore_with_blank_name.py

```diff
@@ -0,0 +1,290 @@
+            kp = self.create_key_pair(
+                tvaultconf.key_pair_name, keypair_cleanup=True)
+            LOG.debug("Key_pair : " + str(kp))
+
+            # create vm
+            vm_id = self.create_vm(key_pair=kp, vm_cleanup=True)
+            LOG.debug("VM ID : " + str(vm_id))
+            time.sleep(30)
+
+            # assign floating ip
+            floating_ip_1 = self.assign_floating_ips(fip[0], vm_id, False)
+            LOG.debug("Assigned floating IP : " + str(floating_ip_1))
+            time.sleep(20)
+
+            # add some data to boot disk...
+            self.data_ops_for_bootdisk(floating_ip_1, data_dir_path[0], 3)
+
+            # Create workload using CLI
+            workload_create = command_argument_string.workload_create + \
+                              " --instance instance-id=" + str(vm_id)
+
+            rc = cli_parser.cli_returncode(workload_create)
+
+            if rc != 0:
+                LOG.debug("Execute workload-create command using cli")
+                raise Exception(
+                    "Execute workload-create command using cli")
+            else:
+                reporting.add_test_step(
+                    "Execute workload-create command using cli",
+                    tvaultconf.PASS)
+
+            time.sleep(10)
+
+            # check workload status
+            wid = query_data.get_workload_id_in_creation(tvaultconf.workload_name)
+            LOG.debug("Workload ID: " + str(wid))
+            if (wid is not None):
+                self.wait_for_workload_tobe_available(wid)
+                if (self.getWorkloadStatus(wid) == "available"):
+                    LOG.debug("Create workload using cli passed.")
+                    reporting.add_test_step(
+                        "Create workload using cli", tvaultconf.PASS)
+                else:
+                    LOG.debug("Create workload using cli failed.")
+                    raise Exception("Create workload using cli")
+            else:
+                LOG.debug("Create workload using cli failed.")
+                raise Exception("Create workload using cli")
+
+            # take full snapshot
+            snapshot_id = self.create_snapshot(wid, is_full=True)
+
+            # Add some more data to files on VM
+            self.data_ops_for_bootdisk(floating_ip_1, data_dir_path[1], 3)
+
+            ### Incremental snapshot ###
+            incr_snapshot_id = self.create_snapshot(wid, is_full=False)
+
+
+            self.wait_for_workload_tobe_available(wid)
+            snapshot_status = self.getSnapshotStatus(wid, snapshot_id)
+
+            if (snapshot_status == "available"):
+                LOG.debug("Full snapshot created.")
+                reporting.add_test_step("Create full snapshot", tvaultconf.PASS)
+            else:
+                LOG.debug("Full snapshot creation failed.")
+                raise Exception("Create full snapshot")
+
+            reporting.test_case_to_write()
```

Comment:
What test case are you adding to report ?

Reply:
I removed this test_case_to_write() as kept only on restore cases.
Adding updated changes.